includes/discovery/sensors/temperature/beagleboard.inc.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * Temperature Sensors for BeagleBoard
+ * Requires snmp extend agent script from librenms-agent
+ */
+
+use Illuminate\Support\Str;
+use LibreNMS\Config;
+
+$type = "beagleboardTemp";
+
+// $oid = 'NET-SNMP-EXTEND-MIB::nsExtendOutLine."beagleboard"';
```

Comment:
could you test this again? We hopefully fixed this

Reply:
Sure - updated, re-pushed. Let's see! 😄.

Was there an issue with the community string?

Reply:
No joy 😞.